components/blitz/src/ome/services/blitz/impl/RoiI.java

```diff
@@ -1,7 +1,7 @@
 /*
  *   $Id$
  *
- *   Copyright 2009 Glencoe Software, Inc. All rights reserved.
+ *   Copyright 2009-2016 Glencoe Software, Inc. All rights reserved.
```

Comment:
No, that was me; just became a habit to adjust copyright date. Will revert that.